prime.cpp

```diff
@@ -0,0 +1,31 @@
+#include <iostream>
+using namespace std;
+
+int main() {
+
+  int i, n;
+  bool is_prime = true;
+
+  cout << "Enter a positive integer: ";
+  cin >> n;
+
+  // 0 and 1 are not prime numbers
+  if (n == 0 || n == 1) {
```

Comment:
As per the [definition](https://math.stackexchange.com/a/1002463) of prime numbers. Generally we don't consider numbers less than 1 to be prime.
```suggestion
  if (n <= 1) {
```